samples/web-nodejs-sample.devfile

```diff
@@ -0,0 +1,27 @@
+specVersion: 0.0.1
+name: nodejs-sample
+projects:
+  - name: nodejs-sample
+    source:
+      type: git
+      location: 'https://github.com/che-samples/web-nodejs-sample.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+commands:
+  - name: start
+    actions:
+      - type: exec
+        tool: theia-editor
+        command: cd /projects/nodejs-sample/app && node app.js
+        workdir: /projects/nodejs-sample/app
+  - name: test
+    actions:
+      - type: exec
+        tool: theia-editor
+        command: curl ${HOSTNAME}:3000
+        workdir: /projects/nodejs-sample/
```

Comment:
Does it really matter where `curl` command will be executed?

Reply:
no